### PR TITLE
Document PageRank algo convergence condition 

### PR DESCRIPTION
**Pull request by Qudirah**
Adding documentation for #4792

### CODE REVIEW

**Comment by Qudirah** [comment]
@rossbar what do you think?

File: networkx/algorithms/link_analysis/pagerank_alg.py [hunk 1]
```diff
@@ -45,6 +45,10 @@ def pagerank(
     tol : float, optional
       Error tolerance used to check convergence in power method solver.
 
+      Note
+      -----
+      Convergence condition is N * tol
```
Comment (rossbar) [inline networkx/algorithms/link_analysis/pagerank_alg.py:50]: There's no indication what N is in this context.

I'd recommend copying the relevant sentence from the `Notes` section on convergence.

**Review by rossbar — CHANGES_REQUESTED** [review]
We'll want to remove the `Note` and the underlines, as that is a special keyword for the docstring format. I'm okay with adding this info to the parameter description, though it's worth mentioning that it's already explicitly addressed in the actual `Notes` section.

File: networkx/algorithms/link_analysis/pagerank_alg.py (hunk 1 above)
Comment (Qudirah) [reply networkx/algorithms/link_analysis/pagerank_alg.py:50]: got it.

Comment (Qudirah) [reply networkx/algorithms/link_analysis/pagerank_alg.py:50]: @rossbar what do you think of this?

"Convergence condition is N * tol where N is len(G). See notes for more explanation. "

Comment (rossbar) [reply networkx/algorithms/link_analysis/pagerank_alg.py:50]: This is of course subjective, but I still don't think that's as clear as what's already in the docstring - I'd really advocating just using the existing wording!

```suggestion
      The iteration will stop after a tolerance of ``len(G) * tol`` is reached.
```

Comment (Qudirah) [reply networkx/algorithms/link_analysis/pagerank_alg.py:50]: Alright. Understood.

Comment (Qudirah) [reply networkx/algorithms/link_analysis/pagerank_alg.py]: @rossbar what do you think?

**Comment by Qudirah** [comment]
@jarrodmillman have a look at this

**Review by MridulS — APPROVED** [review]
Thanks for this @Qudirah and sorry for the delay in review :)

**Review by rossbar — APPROVED** [review]
Thanks @Qudirah !

**Pull request merged**